tests/test_textcat.py

```diff
@@ -0,0 +1,96 @@
+from pathlib import Path
+from typing import Callable, List
+
+import pytest
+import spacy
+
+from recipes.openai import OpenAISuggester, get_api_credentials, load_template
+from recipes.openai_textcat import DEFAULT_PROMPT_PATH, make_textcat_response_parser
+
+
+def make_suggester(
```

Comment:
Is this function used in this file? I couldn't find a use after CMD+F'ing.

Reply:
I kept a refactored version of this function [here](https://github.com/explosion/prodigy-openai-recipes/blob/main/tests/utils.py#L7), ideally for future use. But yeah, I can just add that in when I write the other refactor.